portfolio/src/main/webapp/index.html

```diff
@@ -188,6 +188,26 @@ <h1 class = "arrow">Education</h1>
             <div class = "row">
                 <div class = "col-md-12">
                     <div class = "education">
+                        <!-- Education Box 0 -->
+                        <div class = "col-md-4 wp2 item">
+                            <div class = "icon">
+                                <i class = "fa fa-pencil"></i><!-- Icon -->
```

Comment:
`<i>` tags aren't really used much anymore. They're for stand-out or idiomatic text and the alternatives listed here are typically the inline elements used instead - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i

Reply:
Thank you for letting me know about this🙏🙏🙏